lib/tentacat/hooks.ex

```diff
@@ -49,15 +50,21 @@ defmodule Tentacat.Hooks do
   end
 
   @doc """
-    hook_body = %{
-      "name" => "web",
-      "active" => true,
-      "events" => [ "push", "pull_request" ],
-      "config" => {
-        "url" => "http://example.com/webhook",
-        "content_type" => "json"
-      }
+  Update a hook.
+
+  Update body example:
+
+  ```elixir
+  hook_body = %{
+    "name" => "web",
+    "active" => true,
+    "events" => [ "push", "pull_request" ],
+    "config" => {
+      "url" => "http://example.com/webhook",
+      "content_type" => "json"
     }
+  }
+  ```
```

Comment:
hmm, ideally these and similar changes should go into the Example block for better rendering of the same by ExDoc

Reply:
I see. Maybe for another PR? 

FYI, we've to explicitly specify the code block instead of 4-spaces indentation as the commit body example in https://github.com/edgurgel/tentacat/blob/master/lib/tentacat/commits.ex#L64 cannot render correct.

Reply:
ah yes, it can be another PR.
> FYI, we've to explicitly specify the code block instead of 4-spaces indentation as the commit body example in https://github.com/edgurgel/tentacat/blob/master/lib/tentacat/commits.ex#L64 cannot render correct.

True, if it goes into Example block, then exdoc should render it correct?

Reply:
> ah yes, it can be another PR.
> 
> > FYI, we've to explicitly specify the code block instead of 4-spaces indentation as the commit body example in https://github.com/edgurgel/tentacat/blob/master/lib/tentacat/commits.ex#L64 cannot render correct.
> 
> True, if it goes into Example block, then exdoc should render it correct?

@sreecodeslayer Nope. Not really.

![image](https://user-images.githubusercontent.com/134518/125181663-a558b700-e239-11eb-987b-8648c7e3f057.png)